apps/api/src/app/config/__tests__/utilities/buildMockConfigService.ts

```diff
@@ -9,6 +9,18 @@ export default (configOverrides: ConfigOverrides, envFilePath: string) => {
     const realConfig = dotenv.parse(fs.readFileSync(envFilePath));
 
     const mockedConfig = {
+        NODE_PORT: process.env.NODE_PORT,
```

Comment:
I'm going to investigate whether this was actually necessary, and if so, find a more extensible way to do this.